prow/clonerefs/run.go

```diff
@@ -108,19 +110,21 @@ func (o Options) Run() error {
 	return nil
 }
 
-func addHostFingerprints(fingerprints []string) error {
-	path := filepath.Join(os.Getenv("HOME"), ".ssh", "known_hosts")
+func addHostFingerprints(fingerprints []string) (string, error) {
+	// The clonerefs image uses Alpine, where a /tmp folder always exists
+	path := filepath.Join("/tmp", "known_hosts")
 	f, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
 	if err != nil {
-		return fmt.Errorf("could not create/append to %s: %v", path, err)
+		return "", fmt.Errorf("could not create/append to %s: %v", path, err)
 	}
 	if _, err := f.Write([]byte(strings.Join(fingerprints, "\n"))); err != nil {
-		return fmt.Errorf("failed to write fingerprints to %s: %v", path, err)
+		return "", fmt.Errorf("failed to write fingerprints to %s: %v", path, err)
 	}
 	if err := f.Close(); err != nil {
-		return fmt.Errorf("failed to close %s: %v", path, err)
+		return "", fmt.Errorf("failed to close %s: %v", path, err)
 	}
-	return nil
+	logrus.Infof("Updated known_hosts in file: %s", path)
+	return fmt.Sprintf("GIT_SSH_COMMAND=/usr/bin/ssh -o UserKnownHostsFile=%s", path), nil
```

Comment:
Should we look up (`exec.LookPath`) where SSH is?

Reply:
Good idea, done.